README.md

```diff
@@ -47,3 +30,200 @@ terms of [OCA](http://oracle.com/technetwork/goto/oca).
 - [METRO project](https://javaee.github.io/metro)
 - [JSR 222](https://jcp.org/en/jsr/detail?id=222)
 - [Download standalone distribution](https://repo1.maven.org/maven2/com/sun/xml/bind/jaxb-ri/2.3.0/jaxb-ri-2.3.0.zip)
+
+## Build instructions
+JavaSE 11 is required to build JAXB.
+Simply run `mvn clean install`.
+
+There are three projects inside JAXB repository which are JAXB dependencies, but are not part
+of the main build:
+
+- **jaxb-ri/codemodel**:
+Codemodel is an independent library for java code generation, which is released separately and integrated in JAXB. 
+Besides JAXB it is used in [istack-commons-project](https://github.com/eclipse-ee4j/jaxb-istack-commons), 
+which in turn is a dependency of JAXB
+
+- **jaxb-ri/xsom**:
+XML Schema Object Model (XSOM) is a Java library that allows applications to easily parse XML Schema
+documents and inspect information in them. It is released separately because it is a dependency of 
+[jaxb-fi](https://github.com/eclipse-ee4j/jaxb-fi),
+which in turn is a dependency of JAXB
+
+- **jaxb-ri/external**:
+RNGOM and relaxng-datatype under jaxb-ri/external are tools for working with RelaxNG. It is released separately because it is a dependency of 
+several jaxb-ri projects including jaxb-ri/xsom above.
+
+##Using JAXB with Maven 
+
+###Maven coordinates for JAXB artifacts
+
+-   **java.xml.bind:jaxb-api** API classes for JAXB. Required to compile
```

Comment:
shouldn't this be changed to 'jakarta.xml.bind:jakarta.xml.bind-api'?

Reply:
Good catch.